.github/workflows/tests.yml

```diff
@@ -61,7 +56,11 @@ jobs:
         python -m pip install --upgrade pip
         pip install tox
     - name: Test
+      if: ${{ !startsWith(runner.os, 'Mac') }}
       run: tox -e ${{ matrix.config[1] }}
+    - name: Test (macOS)
+      if: ${{ startsWith(runner.os, 'Mac') }}
+      run: tox -e ${{ matrix.config[1] }}-universal2
```

Comment:
I think this causes the failures on macos since this environment isn't defined in tox.ini?

Reply:
Yes it does, I know.